lib/irc-view.coffee

```diff
@@ -51,12 +52,14 @@ class IrcView extends ScrollView
 
     msg = $('<span>')
     un.addClass 'msg'
+    autolinker = new Autolinker(newWindow: false)
+    linkedText = autolinker.link(message)
     if atom.config.get('irc.evalHtml')
       msg.addClass 'html'
-      msg.html(util.format '%s', message)
+      msg.html(util.format '%s', linkedText)
     else
       msg.addClass 'txt'
-      msg.text(util.format '%s', message)
+      msg.text(util.format '%s', linkedText)
```

Comment:
@aaronholla Also, the `<a/>` tag that is generated by autolinker wouldn't render since it is a `.text` call.